backend/src/main/java/org/fleetassistant/backend/auth/AuthenticationController.java

```diff
@@ -0,0 +1,30 @@
+package org.fleetassistant.backend.auth;
+
+import lombok.RequiredArgsConstructor;
+import org.fleetassistant.backend.auth.models.AuthenticationRequest;
+import org.fleetassistant.backend.auth.models.AuthenticationResponse;
+import org.fleetassistant.backend.auth.models.RegisterRequest;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("api/v1/auth/")
+@RequiredArgsConstructor
+public class AuthenticationController {
+    private final AuthenticationService authenticationService;
+
+    @PostMapping("/register")
+    public ResponseEntity<AuthenticationResponse> register(@RequestBody RegisterRequest request) {
+        AuthenticationResponse registerResponse = new AuthenticationResponse(authenticationService.register(request));
+        return ResponseEntity.ok(registerResponse);
+    }
+
+    @PostMapping("/authenticate")
+    public ResponseEntity<AuthenticationResponse> authenticate(@RequestBody AuthenticationRequest request) {
+        AuthenticationResponse authenticationResponse = new AuthenticationResponse(authenticationService.authenticate(request));
+        return ResponseEntity.ok(authenticationResponse);
+    }
+}
```

Comment:
provide controller method to get user data in response body (with no request body)